components/Editor.vue

```diff
@@ -13,6 +13,7 @@
           <div ref="editorElement">
             <ckeditor
               v-if="editor && config"
+              :key="annotationsSidebarInline"
```

Comment:
this forces re-render of the editor to receive the updated config when changing between mobile and desktop layout